gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/structure/util/ComparatorsTest.java

```diff
@@ -18,20 +18,20 @@
  */
 package org.apache.tinkerpop.gremlin.structure.util;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
There are a lot of these kinds of changes in the pull request where you just change the import order. I imagine that change was done without your knowing by your IDE. Could you please revert those sorts of changes as:

1. It's the opposite of our style - we tend to typically place static imports last and "java.*" imports after other package imports
2. It will simplify this diff for purpose of review.

Thanks.

Reply:
Restored the original order of imports. Please review.

Reply:
lgtm - thank you for doing that.